src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -7191,36 +7197,37 @@ def test_no_halo_exchange_annex_dofs(tmpdir, f90, f90flags, monkeypatch,
 
 
 def test_annexed_default():
-    ''' test that we do not compute annexed dofs by default '''
-    import psyclone.config
-    assert not psyclone.config.COMPUTE_ANNEXED_DOFS
+    ''' Test that we do not compute annexed dofs by default (i.e. when
+    using the default configuration file). '''
+    from psyclone import configuration
+    _config = configuration.Config(config_file=DEFAULT_CFG_FILE)
+    assert not _config.compute_annexed_dofs
 
 
 def test_haloex_not_required(monkeypatch):
     '''The dynamic halo exchange required() logic should always return
     False if read dependencies are to annexed dofs and
-    config.COMPUTE_ANNEXED_DOFS is True, as they are computed by
+    Config.compute_annexed_dofs is True, as they are computed by
     default when iterating over dofs and kept up-to-date by redundant
     computation when iterating over cells. However, it should return
     True if there are no previous write dependencies and
-    config.COMPUTE_ANNEXED_DOFS is False, as a previous writer may
+    Config.compute_annexed_dofs is False, as a previous writer may
     have iterated over dofs and only written to its own dofs, leaving
     the annexed dofs dirty. This test checks these two cases. Note the
     former case should currently never happen in real code as a halo
     exchange would not be added in the first place.
     '''
-    import psyclone.config
-    monkeypatch.setattr(psyclone.config, "COMPUTE_ANNEXED_DOFS", False)
+    monkeypatch.setattr(_CONFIG, "_compute_annexed_dofs", False)
```

Comment:
Perhaps ignore my suggestion as 3 uses is on the edge, as long as there are no side effects.